tasks/tests/test_view_all_tasks.py

```diff
@@ -34,35 +34,36 @@ def test_authenticated_user_tasks_view(self, client, test_db, employee_flag):
         else:
             assert all(task.assignee.team == user.team for task in response.context['tasks'])
 
-    def test_filter_task_by_priority(self, client, test_db):
+    def test_filter_task_by_priority_and_status(self, client, test_db):
         user = test_db[1][0]
         client.login(username=user.user.username, password=DEFAULT_VALID_PASSWORD)
-        response = client.post('/tasks/', data={'priority': 'Low'})
-        assert all(task.priority == Priority.LOW for task in response.context['tasks'])
+        response = client.post('/tasks/', data={'priority': 'Low', 'status': 'Backlog'})
+        assert all(task.priority == Priority.LOW and task.status == Status.BACKLOG for task in response.context['tasks']
+                   )
 
-    @pytest.mark.parametrize('user_type, priority_text',
+    @pytest.mark.parametrize('user_type, priority_text, status_text',
                              [
-                                 (1, 'Low'),
-                                 (1, 'Medium'),
-                                 (1, 'High'),
-                                 (1, 'Critical'),
-                                 (2, 'Low'),
-                                 (2, 'Medium'),
-                                 (2, 'High'),
-                                 (2, 'Critical')
+                                 (1, 'Low', ""),
+                                 (1, 'Medium', ""),
+                                 (1, 'High', ""),
+                                 (1, 'Critical', ""),
+                                 (2, 'Low', ""),
+                                 (2, 'Medium', ""),
+                                 (2, 'High', ""),
+                                 (2, 'Critical', "")
                              ],
                              ids=[
-                                'employee - Low',
-                                'employee - Medium',
-                                'employee - High',
-                                'employee - Critical',
-                                'manager - Low',
-                                'manager - Medium',
-                                'manager - High',
-                                'manager - Critical'
+                                 'employee - Low',
+                                 'employee - Medium',
+                                 'employee - High',
+                                 'employee - Critical',
+                                 'manager - Low',
+                                 'manager - Medium',
+                                 'manager - High',
+                                 'manager - Critical'
                              ]
                              )
-    def test_filter_task_by_priority_randomized(self, client, test_db, priority_text, user_type):
+    def test_filter_task_by_priority_not_randomized(self, client, test_db, priority_text, user_type, status_text):
```

Comment:
Since it is as every test method (not randomize) maybe you can give up on the suffix of the method name and name it like:
```suggestion
    def test_filter_task_by_priority(self, client, test_db, priority_text, user_type, status_text):
```